### ISSUE TITLE
Text change on License and additional terms of use form

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#598]
1. Please change the text at the top of the page to an H2: "License and additional terms of use"
2. The line beneath the title should say "License" in bold Paragraph and have a link next to it "Get help selecting a license"
3. "Additional terms of use" should be bold Paragraph.
